src/app/static/src/app/store/adrUpload/actions.ts

```diff
@@ -30,21 +32,54 @@ export const actions: ActionTree<ADRUploadState, RootState> & ADRUploadActions =
                         const metadata = response.data;
                         const schemas = rootState.adr.schemas!;
 
-                        const uploadFiles = {
-                            outputZip: constructUploadFile(
+                        const uploadFiles: Dict<UploadFile> =  {
+                            outputZip: constructUploadFileWithResourceName(
                                 metadata,
                                 0,
                                 schemas.outputZip,
                                 `${project.name}_naomi_outputs.zip`,
-                                "uploadFileOutputZip"),
-                            outputSummary: constructUploadFile(
+                                "uploadFileOutputZip",
+                                `${project.name} Naomi Outputs`),
```

Comment:
We should revisit the resource names when when we do https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-2421 - should be able to avoid renaming the resources when we update them.